pkg/pthi/commands.go

```diff
@@ -39,6 +40,61 @@ func NewCommand() Command {
 	}
 }
 
+func GetOSIPAddress(results []uint8) (uint32, error) {
```

Comment:
The PTHI package really is meant only for calls that interact with PTHI. Please move this to the internal folder. Along with that, we don't really need to convert the IP into a uint32. The end intention is display this on the command line so why bother converting it to uint32 and change the endian and all, just to convert it back? There no reason to emulate AMT in this regard since this data comes from the OS.